python/py_src/sudachipy/command_line.py

```diff
@@ -17,9 +17,11 @@
 import logging
 import os
 import sys
+from pathlib import Path
 
 from . import __version__
 from . import Dictionary, SplitMode
+from . import sudachipy
```

Comment:
How about to re-import `build_system_dic` and `build_user_dic` in `sudachipy/__init__.py` so that we can import them by `sudachipy.build_~` (instead of `sudachipy.sudachipy.build_~`)?

Reply:
I assumed that we do not expose them (maybe yet) as public API, so didn't do that. Should we expose them then?

Reply:
I guess it's better to provide API at some point, for the use case like preprocess data and build dictionary in a single script.
But if you have some plan to change API around dictionary, we can postpone it.

Reply:
It is sure useful to provide ways of building dictionaries as API, but I am not so sure about the current form (which is certainly usable though).

I will make this an issue for the future work.